ReSwift/CoreTypes/StoreType.swift

```diff
@@ -37,15 +37,6 @@ public protocol StoreType: DispatchingStoreType {
      state in this store changes.
 
      - parameter subscriber: Subscriber that will receive store updates
-     */
-    func subscribe<S: StoreSubscriber>(_ subscriber: S) where S.StoreSubscriberStateType == State
```

Comment:
I would be inclined to leave this in place since we can't put default parameter values in a protocol. Otherwise areas that use `StoreType` as a generic can't use `subscribe` without manually specifying a `nil` selector.